index.html

```diff
@@ -0,0 +1,108 @@
+  </head>
+
+<body>
+  <div class="container">
+    <header id="header">
+        <div id="logo">
+            <img id="header-img" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png" alt="">
+          </div> 
+          <div class="nav-bar">
+            <nav id="nav-bar">
+                <ul>
+                    <li><a class="nav-link" href="#features">Features</a></li>
+                    <li> <a class="nav-link" href="#how-it-works">How It Works</a></li>
+                    <li><a class="nav-link" href="#pricing">Pricing</a></li>
+                </ul>
+
+            </nav>
+          </div>
+    </header>
+    
+    <div class="second-header">
+      <h2>
+        Handcrafted, home-made masterpieces
+      </h2>
+
+    
+    <form id="form" action="https://www.freecodecamp.com/email-submit">
+
+        <input name="email" id="email" type="email" placeholder="Enter your email address" required="">
+        <input id="submit" type="submit" value="Get Started" class="btn">
+    </form>
+  </div>
+    <section id="features">
+      <div class="grid">
+        <div class="icon">
+          <i class='fa fa-3x fa-fire'></i>
+        </div>
+        <div class="desc">
+          <h2>Premium Materials</h2>
+          <p>Our trombones use the shiniest brass which is sourced locally.
+             This will increase the longevity of your purchase.</p>
+        </div>
+      </div>
+      <div class="grid">
+        <div class="icon">
+          <i class='fa fa-3x fa-truck'></i>
+        </div>
+        <div class="desc">
+          <h2>Fast Shipping</h2>
+          <p>We make sure you recieve your trombone as soon as we have finished making it. We also provide free returns if you are not satisfied.</p>
+        </div>
+      </div>
+      <div class="grid">
+        <div class="icon">
+          <i class='fa fa-3x fa-truck'></i>
+        </div>
+        <div class="desc">
+          <h2>Quality Assurance</h2>
+          <p>For every purchase you make, we will ensure there are no damages or faults and we will check and test the pitch of your instrument.</p>
+        </div>
+      </div>
+
+    </section>
+    <section id="how-it-works">
+      <div class="grid">
+        <div class="icon">
+          <i>
+           
+          </i>
+        </div>
+        
+      </div>
+      <div class="grid">
+        <div class="icon">
+          <i ></i>
+        </div>
+      </div>
+
+    </section>
+    <section id="pricing">
+      <div class="grid">
+        <div class="icon">
+          <i ></i>
+        </div>
+      </div>
+
+    </section>
+    <iframe id="video" height="400" width="600"src="https://www.youtube-nocookie.com/embed/y8Yv4pnO7qc?rel=0&controls=0&showinfo=0" frameborder="0" allowfullscreen></iframe>
+   
+  </div>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+
+    <script src="js/index.js"></script>
```

Comment:
Redundant script tags. No JS file is added to project !

Reply:
Please ensure that your comments are clear and easy to understand. For Example:
A good example should be: `Redundant script tags. Please consider removing the JS link tag as you do not have any JS file added to the project!`

Reply:
You have some Redundant script tags here. Why don't you remove the JS link tags, since there is no JS file added to the project?